modules/node_modules/@ncigdc/components/Aggregations/index.js

```diff
@@ -51,3 +51,20 @@ export const CheckedLink = styled(Link, ({
     color: ({ theme }) => theme.success,
   },
 }));
+
+export const facetFieldDisplayMapper = field => {
+  const map = {
+    'genes.is_cancer_gene_census': 'Curated Gene Set',
+    'genes.symbol': 'Gene Symbol',
+  };
+  return map[field] || field;
+};
+
+export const facetValueDisplayMapper = (field, value) => {
+  const map = {
+    'genes.is_cancer_gene_census': { 1: 'Cancer Gene Census' }, // TODO: 🙏 this needs to change in the data
+  };
+
+  const innerMap = map[field];
+  return innerMap ? map[field][value] || value : value;
```

Comment:
`innerMap && innerMap[value] || value;` would work I think

this is probably fine but I can imagine a scenario where this would break.. if innerMap was not an object but a string.. and value was a string property.